de/debug/simulation_debugging.md

```diff
@@ -31,16 +31,12 @@ make px4_sitl_default jmavsim___valgrind
 
 ## Start combinations
 
-SITL can be launched with and without debugger attached and with either jMAVSim or Gazebo Classic as simulation backend. This results in the start options below:
+SITL can be launched with and without debugger attached with Gazebo Classic as simulation backend. This results in the start options below:
```

Comment:
See this file path is /de. That means it is a translation file. Unfortunately that means this can't be merged. We will need to change the same file in the /en path.

Reply:
thanks for explaining that to me. I was slightly confused by that. Will back it out